src/ui/components/constants.ts

```diff
@@ -8,17 +8,25 @@ export const STATUSES = {
   waiting: 'waiting',
 }
 
-export const FIELDS = {
-  [STATUSES.active]: [
-    'attempts',
-    'data',
-    'id',
-    'name',
-    'opts',
-    'progress',
-    'timestamps',
-  ],
-  [STATUSES.completed]: [
+export type Status = keyof typeof STATUSES
+
+export type Field =
+  | 'attempts'
+  | 'data'
+  | 'id'
+  | 'name'
+  | 'opts'
+  | 'progress'
+  | 'timestamps'
+  | 'delay'
+  | 'failedReason'
+  | 'retry'
+  // REVIEW: this one is in none of the statuses, but we have a handler for it?
```

Comment:
Unclear. Open for suggestions on what to do 🙂

Reply:
I'm not sure if I'm missing anything in this one. As far as I understand, we handle the Fields from their mapping from `Status` inside `FIELDS`, but no `Status` is mapping to the `finish` field.
As far as I understand, the line 128 in `src/ui/components/Queue.tsx` is pretty much dead code. Or is it there for the eventful occasion where we'll want to add support to this field? 🤔

Reply:
no plans, for now, kill it